README.Rmd

```diff
@@ -0,0 +1,90 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-",
+  out.width = "100%"
+)
+```
+
+# sourmashconsumr
+
+<!-- badges: start -->
+[![R-CMD-check](https://github.com/Arcadia-Science/sourmashconsumr/actions/workflows/check-standard.yaml/badge.svg)](https://github.com/Arcadia-Science/sourmashconsumr/actions/workflows/check-standard.yaml)
+[![Codecov test coverage](https://codecov.io/gh/Arcadia-Science/sourmashconsumr/branch/main/graph/badge.svg)](https://app.codecov.io/gh/Arcadia-Science/sourmashconsumr?branch=main)
+<!-- badges: end -->
+
+The goal of sourmashconsumr is to parse, analyze, and visualize the outputs of the [sourmash python package](https://sourmash.readthedocs.io/en/latest/).
+The sourmashconsumr package is still under active development.
+
+## Installation
+
+You can install the development version of sourmashconsumr from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("devtools")
+devtools::install_github("Arcadia-Science/sourmashconsumr")
+```
+
+Eventually, we hope to release sourmashconsumr on CRAN and to provide a bioconda package. 
+We'll update these instructions once we've done that.
+
+## Usage
+
+See the vignette for full instructions on how to run the sourmashconsumr package (coming soon!).
+
+To access the functions in the sourmashconsumr package, you can load it with:
+
+```{r example}
+library(sourmashconsumr)
+```
+
+The sourmashconsumr package contains a variety of functions to work with the outputs of the sourmash pythong package.
+`read*` functions read and parse the outputs of sourmash into data frames.
+`plot*` functions visualize the sourmash outputs.
+`from_*to_*` functions convert sourmash data frames into the formats of other popular R packages (e.g. phyloseq).
+sourmashconsumr also contains a few utility functions that work with other datatype (at this time, mostly upset data frames).
+See a complete list of functions below, sorted by data type that the function operates on:
```

Comment:
does this mean they're sorted by the function inputs or what exactly does "data type that the function operates on" mean?

Reply:
```suggestion
See a complete list of functions below, sorted by sourmash output that the function operates on:
```

i changed the wording! They're mostly sorted by the typed of sourmash output that the function was written for, but then there's all the upset utilities and those aren't sourmash specific, but I think maybe it's ok and the wording above will make it more clear